docs/how-to-pr.adoc

```diff
@@ -0,0 +1,43 @@
+= Workflow to make your first Pull Request
+
+Assuming that you already have an Eclipse User account and signed the Eclipse Contribution Agreement, the following is how you can create PR's for Jakarta EE related project:
+
+- If it's the first time you are going to work on the repository: Fork into your account the repository https://github.com/jakartaee/collateral
+
+- Download (you can use the Github UI option `Open With Github Desktop`) your new fork into your computer.
+
+- In your local repository, create a branch for they contribution, example:
+
+    git checkout -b jakartaEEandMP
+
+
+- Set your upstream to your origin:
+
+    git push --set-upstream origin jakartaEEandMP
+
+- Add, edit or perform the changes required in your contribution.
+
+- Stage your new files:
+
+   git add .
+
+- Commit your changes locally:
+
+    commit -m "Added Synergy JakartaEE and MicroProfile artwork."
```

Comment:
==> 
  git commit -m "Added Synergy JakartaEE and MicroProfile artwork."

Reply:
fixed in #a618ce9